This patch fixes the parallel make issue on a 40 way build system

Nitin A Kamble <nitin.a.kamble@intel.com>
2011-02-16

Index: perl-5.12.2/Makefile.SH
===================================================================
--- perl-5.12.2.orig/Makefile.SH	2010-09-06 16:30:32.000000000 -0700
+++ perl-5.12.2/Makefile.SH	2011-02-16 16:21:30.744143773 -0800
@@ -198,6 +198,8 @@
 $this_target: uni.data" ;;
 	Text/ParseWords) extra_dep="$extra_dep
 $this_target: lib/auto/Scalar/Util.$dlext" ;;
+	POSIX) extra_dep="$extra_dep
+$this_target: lib/auto/Cwd/Cwd.$dlext" ;;
     esac
 done
 
